src/components/OfficeBlocks/OfficeBlocksComponent.jsx

```diff
@@ -48,14 +52,27 @@ class OfficeBlocksComponent extends React.Component {
     this.props.resetMessage();
   };
 
+  handleDropdownChange = (e, data) => {
+    this.props.loadCentreOfficeBlocks(data.value);
+  };
+
   getTotalPages = () => Math.ceil(this.props.blockCount / this.state.limit);
 
   render() {
-    const { isLoading, blockList, error, resetMessage, entity } = this.props;
+    const { isLoading, blockList, error, resetMessage, entity, locationList } = this.props;
     const hasLocations = !isEmpty(blockList);
     const showStatus = error;
     const showAction = entity === 'office-blocks';
     const showNotFound = !isLoading && !hasLocations && !showStatus;
+    const options = [];
+    locationList.forEach((location, index) => {
+      const locObj = {
+        key: index,
+        text: location.name,
+        value: location.id
+      };
+      options.push(locObj);
```

Comment:
Please make `Select Center` The first Option as well. It vanishes and replaces it with the first option once the dropdown is clicked.